js/ui/calendar/ui.calendar.base_view.js

```diff
@@ -178,11 +192,11 @@ const BaseView = Widget.inherit({
         };
     },
 
-    _iterateCells: function(colCount, delegate) {
+    _iterateCells: function(colCount, callback) {
```

Comment:
Get rid of it?

Reply:
Also removed the code duplication when rendering the header row of the month view